transit_odp/static/openapi/avl3.yml

```diff
@@ -36,16 +36,17 @@ paths:
         - name: boundingBox
           in: query
           description: >
-            Limit results to bus location data with vehicle position within the
-            rectangular boundingBox you set using co-ordinates.
-            [minLongitude, minLatitude, maxLongitude, maxLatitude]
+            Limit results to bus location data with vehicle position within the rectangular
+            boundingBox you set using co-ordinates.
+            [minLongitude, minLatitude, maxLongitude, maxLatitude].
+            The default values shown are for the Liverpool area.
           schema:
             type: array
             items:
               type: number
             minItems: 4
             maxItems: 4
-            example: [51.401, 51.509, 0.01, 0.201]
```

Comment:
This bounding box change also needs to be done for '/gtfsrtdatafeed'

Reply:
@surabhika Added these in.